app/helpers/vita_min_form_builder.rb

```diff
@@ -45,6 +45,44 @@ def h1_label_contents(label_text, help_text, optional = false)
     label_text.html_safe
   end
 
+  def vita_min_state_file_select(
+    method,
+    label_text,
+    collection,
+    options = {},
+    &block
+  )
+
+    html_options = {
+      class: "select__element",
+    }
+
+    label_class = options[:label_class] || ""
+    label_class += options[:hide_label] ? " sr-only" : ""
+    formatted_label = label(
+      method,
+      label_contents(
+        label_text,
+        options[:help_text],
+        options[:optional],
+        ),
+      class: label_class,
+      )
+    html_options_with_errors = html_options.merge(error_attributes(method: method))
+
+    html_output = <<~HTML
+          <div class="form-group#{error_state(object, method)}">
+            #{formatted_label}
+            <div class="select">
+              #{select(method, collection, options, html_options_with_errors, &block)}
+            </div>
+            #{errors_for(object, method)}
+          </div>
+        HTML
+
+    html_output.html_safe
+  end
+
```

Comment:
did all this just to add the label_class option, because the label on these select inputs should be bold. and looking across the state file designs, i think whether it will be bold might depend on whether there's a single question or if it's part of a bigger form group, in which case it should be an option. and the vita_min_select function is already taken, but it implements the labels as h1s, which i don't think we want either. so i'm not super in love with this but i think we need something different than what we've got 🤷

Reply:
That sounds frustrating to have to do. I believe it's possible to update the styles with css only, so I don't think this is necessary. We also have [a story](https://www.pivotaltracker.com/story/show/186624747) for updating select elements across all of state file.

Reply:
re-checked the designs and confirmed, some "form-question" labels are bold and others are not, and there's nothing i can see to differentiate them in the code currently without adding a class. so i'm gonna leave this for now, maybe later designs will change or someone will be smarter and we can get rid of it.